packages/calcite-components/src/components/combobox/combobox.scss

```diff
@@ -250,6 +250,17 @@ calcite-chip {
   z-index: var(--calcite-z-index-sticky);
 }
 
+.no-matches {
+  @apply block px-4 py-2;
```

Comment:
Should these use our global css vars for spacing?

Reply:
Agreed, we should be using tokens for this. The padding should also match Combobox Item at each scale.